packages/block-library/src/button/block.json

```diff
@@ -9,6 +9,14 @@
 	"keywords": [ "link" ],
 	"textdomain": "default",
 	"attributes": {
+		"tagName": {
+			"type": "string",
+			"default": "a"
+		},
```

Comment:
This should probably be an `enum` as we only want to allow `a` and `button` elements.

Reply:
I don't think we can actually use `enum`... At least not without changing the [schema for block.json](https://github.com/WordPress/gutenberg/blob/7652de9ed159d4e9698e432c06b83e56777160c8/schemas/json/block.json#L106-L138) 🤔

Reply:
What's the exact reason?

I thought we could use something like this:

https://github.com/WordPress/gutenberg/blob/6e9c1d3dc2e325b000494738dfeeaa66a407395e/packages/block-library/src/paragraph/block.json#L29-L32

Reply:
Ah OK!! That makes sense now... I thought you were talking about making the `type` an `enum` 😅